components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c

```diff
@@ -393,36 +358,27 @@ psa_status_t psa_aead_encrypt(psa_key_handle_t key_handle,
     memcpy(buffer, additional_data, additional_data_length);
     memcpy(buffer + additional_data_length, plaintext, plaintext_length);
 
-
     in_vec[0] = (psa_invec) {
```

Comment:
let's be consistent, move the declartion of in_vec and out_vec to here

Reply:
ok